c/cert/test/rules/ERR30-C/test.c

```diff
@@ -0,0 +1,72 @@
+#include <errno.h>
+#include <limits.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+void f1(const char *c_str) {
+  unsigned long number;
+  char *endptr;
+
+  number = strtoul(c_str, &endptr, 0); // NON_COMPLIANT
```

Comment:
I think (I think) are missing a test for false positives wherein it is permissible to look at the value of `errno` without first comparing it against the return value of the function.